### PR TITLE
Tsconfig zh

### PR DESCRIPTION
**Pull request by Kingwl**
Add 
- copy for zh
- allowSyntheticDefaultImports
- esModuleInterop

They are very confusing options. So I have translated them first.

### CODE REVIEW

File: packages/tsconfig-reference/copy/zh/options/esModuleInterop.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+display: "ES 模块互操作性"
+oneline: "为了便于支持导入 commonjs 模块生成额外的 JS"
+---
+默认情况下（未设置 `esModuleInterop` 或值为 false），TypeScript 像 ES6 模块一样对待 Commonjs/AMD/UMD。这样的行为有两个被证实的缺陷：
```
Comment (laosb) [inline packages/tsconfig-reference/copy/zh/options/esModuleInterop.md]: CommonJS

**Review by laosb — APPROVED** [review]

**Comment by orta** [comment]
Good luck, you can decide yourself when you want to merge 👍🏻

File: packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
-display: "Allow Umd Global Access"
-oneline: "Assume UMD imports are all globally available"
+display: "允许 Umd 全局使用"
```
Comment (crimx) [inline packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md]: Access 应为访问。访问只是一个读的操作，使用带有执行的意思。

File: packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+oneline: "假设 UMD 的导入是全局可用的"
 ---
 
-When set to true, `allowUmdGlobalAccess` lets you access UMD exports as globals from inside module files. A module file is a file that has imports and/or exports. Without this flag, using an export from a UMD module requires an import declaration.
+当 `allowUmdGlobalAccess` 设置为 true 时，将允许你在模块文件中以全局变量的形式使用 UMD 的导出。
```
Comment (crimx) [inline packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md]: Access 访问

File: packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
+模块文件是具有或同时导入、导出的文件。当未设置这个选项时，使用 UMD 模块的导出需要首先导入声明。
 
-An example use case for this flag would be a web project where you know the particular library (like jQuery or Lodash) will always be available at runtime, but you can’t access it with an import.
+一个典型场景是：在一个 Web 项目中， 您知道特定的库（如 jQuery 或 Lodash ）在运行时总是可用的，但您不能在通过导入来使用他们。
```
Comment (crimx) [inline packages/tsconfig-reference/copy/zh/options/allowUmdGlobalAccess.md]: 您知道某个特定的库（如 jQuery 或 Lodash ）在运行时总是可用的，但无法通过导入模块来访问。

**Comment by Kingwl** [comment]
Is everything ok?
I'd like to merge this if no more append after 24:00 UTC+8.

**Comment by orta** [comment]
Sounds cool to me - you can tell the bot to merge when you feel good 👍🏻

**Comment by orta** [comment]
( That said, my recommendation is to delete the English ones - then you know you have the latest versions when you translate them ( #1275 for example just added and amended a bunch )  and https://github.com/microsoft/TypeScript-Website/issues/296 will be accurate on the TODO )

**Comment by Kingwl** [comment]
There's no new changes(only remove other files). And no more append comment.
So (I'm not sure it's work).

LGTM.

**Pull request merged**